e2e/e2e-test-parca.sh

```diff
@@ -0,0 +1,88 @@
+################################################################################
+#
+# This script is meant to be run from the root of this project
+#
+################################################################################
+
+#!/bin/bash
+
+set -euo pipefail
+
+function e2erun() {
+  minikube_cluster_up
+  deploy
+  # minikube_cluster_down genius, spin deploy and delete ugh
+}
+
+# Create local minikube cluster and deploys the dev env for parca and parca agent 
+function minikube_cluster_up() {
+  echo "Spinning up a parca dev cluster"
+  minikube start -p parca-dev \
+    --driver=kvm2 \
+    --kubernetes-version=v1.22.3 \
+    --cpus=4 \
+    --memory=16gb \
+    --disk-size=20gb \
+    --docker-opt dns=8.8.8.8 \
+    --docker-opt default-ulimit=memlock=9223372036854775807:9223372036854775807
+}
+
+# Delete minikube instance
+function minikube_cluster_down() {
+  echo "deleting parca dev cluster"
+  minikube delete -p parca-dev
+}
+
+# Fetches latest binaries for parca and parca agent 
+function deploy() {
+  SERVER_LATEST_VERSION=$(curl -s https://api.github.com/repos/parca-dev/parca/releases/latest | grep -oP '"tag_name": "\K(.*)(?=")' | xargs echo -n)
+  AGENT_LATEST_VERSION=$(curl -s https://api.github.com/repos/parca-dev/parca-agent/releases/latest | grep -oP '"tag_name": "\K(.*)(?=")' | xargs echo -n)
```

Comment:
We shouldn't be needing this. We want to build a container from current branch, no?

Reply:
https://blog.hermes-technology.de/minikube/kubernetes/podman/2020/03/23/use-local-images-in-minikube-3-methods.html

https://minikube.sigs.k8s.io/docs/handbook/pushing/